tests/integ/modin/io/test_read_snowflake.py

```diff
@@ -361,12 +361,13 @@ def test_read_snowflake_with_views(
             with caplog.at_level(logging.DEBUG):
```

Comment:
update the logging capture level to WARNING for testing

Reply:
Done!